msmbuilder/featurizer/featurizer.py

```diff
@@ -18,7 +18,7 @@
 import itertools
 from sklearn.base import TransformerMixin
 from sklearn.externals.joblib import Parallel, delayed
-
+from Bio import SeqIO
```

Comment:
Have a try-except for users that don't have `biopython`. It's not a requirement for MSMBuilder, right?